### DISCUSSION TITLE
How to increase silence-timeout ?

### DISCUSSION THREAD

**Discussion by Chaitanya-Jadhav** [Kaljurand/K6nele#76] [Q&A] [answered]
The app stops recording immediately on silence. From what I have understood is that when there is no speech input the application immediately sends the EOS signal. Is there any configuration parameter to increase silence-timeout time on the application side before sending the EOS signal?

**Comment by Kaljurand** [comment, marked as answer]
On the Kõnele side there is no parameter to request or force a certain timeout, e.g. if you're using the "Kõnele (fast recogition)" service then all the voice activity detection and the decision on EOS is done by an external server. Kõnele is not attempting to "configure" it beyond sending it the audio + some meta info (listed in https://github.com/Kaljurand/K6nele/blob/master/privacy_policy.rst). In case you are running your own server then you can configure it directly, otherwise you're out of luck.

Can you describe your usage more precisely (perhaps with screenshots/screencasts), I'm a bit puzzled about the "immediate" stopping, normally this shouldn't happen (with any server).
